services/tenant-ui/frontend/src/components/issuance/schemas/CreateSchemaForm.vue

```diff
@@ -204,17 +206,19 @@ function jsonToSchema(jsonString: string): SchemaSendRequest | undefined {
     formFields.version = parsed.schema_version;
     return parsed;
   } else {
-    toast.error('The JSON you inputted has invalid syntax');
+    toast.error('Invalid JSON detected');
     return undefined;
   }
 }
 // Form submission
 const submitted = ref(false);
 const handleSubmit = async (isFormValid: boolean) => {
   submitted.value = true;
-
+  console.log('trying to validate');
   try {
+    console.log('is valid?');
     if (!isFormValid) return;
+    console.log('valid');
```

Comment:
Probably don't need these logs

Reply:
Both are just leftovers from testing the colour toggling. I have now removed them. Thanks for the review!